README.md

```diff
@@ -34,10 +34,30 @@ If you just want to play on a server hosted by someone else, you need to follow
 
 ![Client](docs/images/client.jpg)
 
+If you do not wish to execute any programs. Follow the instructions in [this website](https://impostor.github.io/Impostor)
+
+#### Android
+
+##### Android 10 and below.
+1. Go to [this website](https://impostor.github.io/Impostor) **(ON YOUR MOBILE DEVICE)**
+2. Follow its instructions.
+
+##### Android 11.
+1. Connect your phone to a computer. Go to [this website](https://impostor.github.io/Impostor) in your computer to generate a file named `regioninfo.dat`.
+2. Instead of following the website instructions please go to `/sdcard/Android/data/com.innersloth.spacemafia` from the computer's file explorer and paste the file there.
```

Comment:
Why are the instructions written for an sd card? The website has the correct file path for my A11 phone, and your instruction didn't work. 

There are also more steps for Android 11, as your steps didn't let me see the files. Here are the correct steps:

Unlock your phone.
With a USB cable, connect your phone to your computer.
On your phone, tap the "Charging this device via USB" notification.
Under "Use USB for," select File Transfer.
A file transfer window will open on your computer. Use it to drag files.
When you're done, eject your phone from Windows.
Unplug the USB cable.

https://support.google.com/android/answer/9064445?hl=en

Reply:
As weird as it sounds. Android calls its internal storage /sdcard/

Reply:
And if you expect me to babysit you thro all the steps for you to connect your phone then you my friend are not suited to do it

Reply:
Not on my phone.
![image](https://user-images.githubusercontent.com/10859074/99842914-4a514300-2b36-11eb-8dde-4b68bee565c8.png)

Reply:
Im giving a guide on how to connect to Impostor, not to connect your phone to the computer

Reply:
If someone follows these instructions, they will not be able to find the file. Plus the path is wrong.

Reply:
![imagem](https://user-images.githubusercontent.com/7207103/99843620-c31dc280-2b69-11eb-96bb-9c9cbc1eaeda.png)

![imagem](https://user-images.githubusercontent.com/7207103/99843594-b7ca9700-2b69-11eb-85ed-fee5a430e5c1.png)

Reply:
You said what?

Reply:
Well, if people look at it through file explorer, it's not going to say /sdcard/ is it? I guess we can say /sdcard if you are doing it through command line/adb and just say `Go to your phone under This PC or Devices`

Reply:
I literally showed you a screenshot of it on my computer. It is not sdcard for me.

Reply:
OK you know what Imma do a quick fix since I want to see this over

Reply:
Done
It is sdcard, but in the command line and as internal name

Reply:
Your instructions aren't working for me:
![image](https://user-images.githubusercontent.com/10859074/99844140-458d8e80-2b38-11eb-9823-8203f9510360.png)

Reply:
You also have to go to com.innersloth.spacemafia\files not just com.innersloth.spacemafia

Reply:
look. I dont care ok? i already put in a note mentioning that sdcard is internal storage. isnt that enough?

Reply:
If you don't care, why did you make the PR. I showed you evidence android doesn't call it sdcard. The only evidence you have is a discord comment. Your PR is just wrong.

Reply:
Are you legit that dumb?
https://ux.stackexchange.com/questions/102778/should-the-internal-storage-of-a-phone-tablet-be-referred-to-as-sd-card-in-the
do some research and then come back. I can also adb into my phone and show you that too

Reply:
Your evidence is a 3 year old post? Ok here is some evidence for you:

![image](https://user-images.githubusercontent.com/10859074/99845868-f7c65580-2b3a-11eb-9657-02edc6efad2f.png)

![image](https://user-images.githubusercontent.com/10859074/99845977-23494000-2b3b-11eb-9ad6-97fed7526669.png)

Reply:
https://android.stackexchange.com/questions/205430/what-is-storage-emulated-0
what about this?

Reply:
one year old
do you need to keep being fed evidence?

Reply:
Why do you want to die on this hill lol. I followed your steps and they were wrong. I tried navigating to the sdcard folder, but it is no where to be found.

Reply:
Because you dont know the meaning of the /sdcard/ folder.
Whoever doesnt know that just reads the note I put afterwards
whoever doesnt know where they cant find it they will research or try finding a familiar folder named Android
whoever cant find it will ask in the discord
Ill be there waiting

Reply:
Why would you leave ambiguous documentation when I literally tested this with an Android 11 device.

Reply:
Because someone else tested it with an android 11 device and it worked. So its a 50/50

Reply:
I have tested it with two different phone, both running Android 11. On two different Windows 10 devices, the folder "sdcard" never showed up.

Reply:
When I said it worked. I MEANT THE PERSON HAD A BRAIN AND KNEW HOW TO FIND IT

Reply:
I have never seen someone get so mad over a PR haha. I literally showed you a screenshot of the file open. Documentation should be clear. On Windows 10 the file explore opens up with your Android 11 device. The only option is Internal shared storage. Maybe linux is different and should have different instructions. But for Windows 10, this is how it is.

Reply:
I know it is, thats why I added a note on sdcard meaning Internal storage. Can you go and read it again???

Reply:
Also I get mad over you not understanding what you are talking about and still firmly believe about being right even after being shown proof. Internal storage is a friendly name, not the real name

Reply:
Again, this is ambiguous documentation. I have never seen sdcard. Maybe you should do more testing to see when the file explorer will show sdcard vs internal shared storage, instead of trying to get a lazy PR in.

Reply:
I know it appears internal storage. I still prefer to call it by its real name, since the friendly name can vary

Reply:
If you call this a lazy PR Im yet to see one of yours :)

Reply:
Maybe you should research when it will so up as sdcard, because it doesn't on Windows 10. 

Also I do enough of this at work, I would rather just have fun with the api after work.

Reply:
Are you blind.
Do you know the difference between friendly name and real mame???
It wont never appear in friendly uis

Reply:
So why does the documentation include it? If it will never appear?

Reply:
Because it is the real name. And thats the one that matters for me. I put a note in for everyone else

Reply:
What matter to you and what matters to the end user are two different things. If the end users see internal storage, the documentation should say that.

Reply:
And it says. In the note. But you seem to be blind to it.